examples/with-auth/server.js

```diff
@@ -0,0 +1,33 @@
+const express = require('express')
+const session = require('express-session')
+const bodyParser = require('body-parser')
+const next = require('next')
+const auth = require('./auth')
+
+const dev = process.env.NODE_ENV !== 'production'
+const app = next({ dir: '.', dev })
```

Comment:
It's actually taken from the `custom-server-with-express` example. I guess for verbosity of the example.